templates/repo/branch/list.tmpl

```diff
@@ -229,6 +231,9 @@
 	<form class="ui form" action="{{$.Repository.Link}}/settings/rename_branch" method="post">
 		<div class="content">
 			{{.CsrfTokenHtml}}
+			<div class="field default-branch-warning">
+				<span class="text red">{{.locale.Tr "repo.branch.warning_raname_default_branch"}}</span>
```

Comment:
I have to point out again (I have left too many tribal comments for this PR):

here , warning_r[a]name_default_branch, above: warning_r[e]name_default_branch


Please be careful!!!!  And  wolfogre  has told you "And the typo raname needs to be fixed elsewhere as well."

Reply:
Sorry for the mistake, should I create a PR to fix the typo?